src/fuzz_introspector/datatypes/function_profile.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, elem: Dict[Any, Any]) -> None:
         self.edge_count = elem['EdgeCount']
         self.cyclomatic_complexity = elem['CyclomaticComplexity']
         self.functions_reached = utils.load_func_names(elem['functionsReached'], False)
+        self.functions_called = utils.load_func_names(elem['functionsReached'], False)
```

Comment:
This looks to be the same as `functions_reached`? Any reason not to use that?

Reply:
Yes, I duplicate it on-purpose. Because self.functions_reached got further processed in 
https://github.com/ossf/fuzz-introspector/blob/main/src/fuzz_introspector/datatypes/fuzzer_profile.py#L231
for further analysis and the original set of "functionsReached" is lost in progress which is needed to build up the callpath tree for the sink analyser.

Reply:
Can you leave a comment in the code saying this?

Reply:
OK